lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
I checked this thing twice on a windows VM and I can confirm that, on win32 platform, the file mode is the same for files with different permissions. So we have to return a fixed value or consider to use the file extension as distinctive.

![image](https://cloud.githubusercontent.com/assets/5115382/17273260/b0f7cef0-56ae-11e6-92e0-f0c8c6359209.png)

But, 'till win support will be effective, we could support at least *nix system with this feature :)